app/src/main/java/org/simple/clinic/settings/SettingsUpdate.kt

```diff
@@ -10,6 +11,7 @@ class SettingsUpdate : Update<SettingsModel, SettingsEvent, SettingsEffect> {
     return when (event) {
       is UserDetailsLoaded -> next(model.userDetailsFetched(name = event.name, phoneNumber = event.phoneNumber))
       is CurrentLanguageLoaded -> next(model.currentLanguageFetched(event.language))
+      is ChangeLanguage -> justEffect(OpenLanguageSelectionScreenEffect)
```

Comment:
It is fine for us to emit this effect now even though there is no handler because the event is not getting triggered from the UI. The handler will be added in a later commit.